lib/main.js

```diff
@@ -370,6 +375,23 @@ module.exports = class Client {
         );
     }
 
+    async sync() {
+        if (!this.publicFeedUrl || !this.publicBundleUrl) {
+            const { body } = await get(
+                url.resolve(this.buildServerUri, '/sync/')
+            );
+            try {
+                const parsedBody = JSON.parse(body);
+                this.publicFeedUrl = parsedBody.publicFeedUrl;
+                this.publicBundleUrl = parsedBody.publicBundleUrl;
+            } catch (err) {
+                throw new Boom(
```

Comment:
can you use `boomify(err, {message: 'sdasdas'})` so the original error message is not lost?

Reply:
ok fixed